ingestion_tools/scripts/transform_ingestion_configs.py

```diff
@@ -35,8 +36,9 @@ def has_no_sources(data: list[dict[str, Any]] | dict[str, Any]) -> bool:
     return isinstance(data, dict) or not any(row.get("sources") for row in data)
 
 
-def update_config(data: dict[str, Any]) -> dict[str, Any]:
+def update_config_to_v1(data: dict[str, Any]) -> dict[str, Any]:
```

Comment:
TODO - changing this method name breaks the gjensen transformation script.

Reply:
TODO:
- Make sure that this code gets executed by `gjensen_config.py` to make sure gjensen outputs get written as v1.1
- Move these ingestion configs manipulation scripts to `ingestion_tools/scripts/migrations`